playback/src/player.rs

```diff
@@ -651,20 +651,24 @@ impl PlayerTrackLoader {
                 FileFormat::OGG_VORBIS_96,
             ],
         };
-        let format = formats
-            .iter()
-            .find(|format| audio.files.contains_key(format))
-            .unwrap();
 
-        let file_id = match audio.files.get(&format) {
-            Some(&file_id) => file_id,
+        let entry = formats.iter().find_map(|format| {
+            if let Some(&file_id) = audio.files.get(format) {
+                Some((*format, file_id))
+            } else {
+                None
```

Comment:
A bit late, but FWIW, it would be nice to know when a desired format wasn't available?
Something like
 ```
 warn!("<{}> unavailable in format {:?}", audio.name, format);
 ```